c2corg_api/views/area.py

```diff
@@ -47,13 +47,13 @@ def put(self):
         return self._put(Area, schema_area, after_update=update_associations)
 
 
-def insert_associations(area):
+def insert_associations(area, user_id):
```

Comment:
`insert_associations` is called down here:
https://github.com/c2corg/v6_api/pull/177/files/8a2af2cbd823d03305c69d4e3c9611027caabf58..e447928a757b40dde6a4c763044d7cb109b4993c#diff-b906af94797c5db6c394971ac60bf7a4R218

Some methods that use the `after_add` hook require the user id.